acro/utils.py

```diff
@@ -64,15 +68,35 @@ def finalise_json(filename: str, results: dict) -> None:
     results : dict
         Outputs to write.
     """
-    # convert dataframes to json
+
     outputs: dict = copy.deepcopy(results)
-    for _, output in outputs.items():
+
+    absolute_path = os.path.dirname(os.path.dirname(__file__))
+    relative_path = OUTPUT_DIRECTORY
+    path = os.path.join(absolute_path, relative_path).replace("\\", "/")
```

Comment:
Replacing the slashes (also used in the tests) to make it OS independent feels wrong to me --- wouldn't os.path.normpath or similar be safer for future changes?